pkg/shell/base_index.js

```diff
@@ -924,7 +924,9 @@ function CompiledComponents() {
                 };
 
                 // Always first keyword should be page name
-                item.keywords[0].matches.unshift(item.label.toLowerCase());
+                const page_name = item.label.toLowerCase();
```

Comment:
How does it end up being there like 50 times?

Reply:
I don't know, exactly.  I guess we create a lot of `Compiled` objects, maybe.

Reply:
Yes, looks like compiled() is *always* called from `function navigate()` without storing the result anywhere. That looks like a bug.